src/libs/Navigation/AppNavigator/BaseDrawerNavigator.js

```diff
@@ -49,6 +49,18 @@ class BaseDrawerNavigator extends Component {
         };
     }
 
+    componentDidUpdate(prevProps) {
+        if (prevProps.isSmallScreenWidth === this.props.isSmallScreenWidth) {
+            return;
+        }
+        // eslint-disable-next-line react/no-did-update-set-state
```

Comment:
can you please add a new line above this comment 😅

comments should have a new line above them (idk why the linter didn't catch this)

Reply:
@rushatgabhane Sure